ambry-api/src/main/java/com.github.ambry/store/Store.java

```diff
@@ -71,6 +71,13 @@
   Set<StoreKey> findMissingKeys(List<StoreKey> keys) throws StoreException;
 
   /**
+   * Get the corresponding {@link StoreStats} instance for this store.
+   * @return a {@link StoreStats} with stats related information
```

Comment:
returns a {@link StoreStats} instance which can be used to fetch store related stats